app/models/article.rb

```diff
@@ -589,7 +589,6 @@ def skip_indexing?
       (score < Settings::UserExperience.index_minimum_score &&
        user.comments_count < 1 &&
        !featured) ||
-      published_at.to_i < 1_500_000_000 ||
```

Comment:
:wave: Thanks for opening this PR and the original issue here! 

I wasn't immediately clear where this condition came from, so I spent a few minutes digging. It looks like it originated in [this PR](https://github.com/forem/forem/pull/14801/files#diff-24503fd25ed68e6ebceee4951bc4f9b255b197278d4aa9d86ef9d5afe3f26beaR501) from 2021, where it seems to be pretty deliberately de-indexing older articles. (It changes from `featured_number` to the current `published_at` in [this PR](https://github.com/forem/forem/pull/18510/files#diff-e45f0ec3372edbef08d5a1a7563b9e72dbd017794c286d90ca8c2fc2f434a982L117-R116), which was removing `featured_number` from articles anyway.)

I have the sense that some Forems really _do_ want to de-index older content, but some Forems do _not_. I wonder if a way to accommodate both use-cases might be to add a configurable setting?